docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+
+Default branch
+  Name of the default branch of the project, leave it to ``main``.
+
+Edit advanced project options
+  Leave it unchecked, we will make some changes later.
+
+After hitting the "Next" button, you will be redirected to the project home.
+You just created your first project on Read the Docs!
+
+.. figure:: /_static/images/tutorial/rtd-project-home.png
+   :width: 80%
+   :align: center
+   :alt: Project home
+
+   Project home
+
+Checking the first build
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs will try to build the documentation of your project
+right after you create it.
+To see the build logs,
+click on the "Your documentation is building" link on the dashboard,
+or alternatively navigate to the "Builds" page,
+then open the one on top (the most recent one).
+
+If the build has not finished yet by the time you open it,
+you will see a spinner next to a "Installing" or "Building" indicator,
+meaning that it is still in progress.
+
+.. figure:: /_static/images/tutorial/rtd-first-successful-build.png
+   :width: 80%
+   :align: center
+   :alt: First successful documentation build
+
+   First successful documentation build
+
+When the build finishes, you will see a green "Build completed" indicator,
+the completion date, the elapsed time,
+and a link to see the corresponding documentation.
+If you now click on "View docs", you will see your documentation live!
+
+.. figure:: /_static/images/tutorial/rtd-first-light.png
+   :width: 80%
+   :align: center
+   :alt: HTML documentation live on Read the Docs
+
+   HTML documentation live on Read the Docs
+
+Basic configuration changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now proceed to make some basic configuration adjustments.
+For that, click on the "⚙ Admin" button, which will open the Settings page.
+
+First of all, add the following text in the description:
+
+    Lumache (/lu'make/) is a Python library for cooks and food lovers
+    that creates recipes mixing random ingredients.
+
+Then set the project homepage to ``https://world.openfoodfacts.org/``,
+and write ``food, python`` in the list of tags.
+All this information will be shown on your project home.
+
+After that, configure your email so you get a notification if the build fails.
+To do so, click on the "Notifications" link on the left,
+type the email where you would like to get the notification,
+and click the "Add" button.
+After that, your email will be shown under "Existing Notifications".
+
+Trigger a build from a pull request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs allows you to :doc:`trigger builds from GitHub pull requests </pull-requests>`
+and gives you a preview of how the documentation would look like with those changes.
+
+To enable that functionality, first click on the "Advanced Settings" link on the left
+under the "⚙ Admin" menu, check the "Build pull requests for this project" checkbox,
+and click the "Save" button at the bottom of the page.
```

Comment:
We should definitely make this easier to do :(

Reply:
Or evaluate the cost or make it opt-out? I hypothesize that most projects use only the default branch anyway (no extra $ on our side), and those who do use pull requests would really appreciate having this feature but they might not know about it.

Reply:
Yea, I could be convinced, tho I worry it's the same as pdf/epub, where if it's on by default most users still won't use it (and might get annoyed by it). I think we need to promote it more, either way.